src/main/java/org/agh/electer/core/domain/subject/choice/SubjectChoice.java

```diff
@@ -0,0 +1,22 @@
+package org.agh.electer.core.domain.subject.choice;
+
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+@Builder
+public class SubjectChoice {
+    @NotNull
+    private Id id;
+
+    @NotNull
+    private Priority priority;
+
+    @NotNull
+    private SubjectId subject;
```

Comment:
Jesli student bedzie mial liste subjectchoiców to jego id tutaj nie dodajemy a zostawiamy subjectID , lub na odwrót jesli to subject bedzie mial liste subjectChoiców